.github/workflows/management-changelog.yml

```diff
@@ -0,0 +1,103 @@
+###
+# @format
+# -----
+# Project: zenstack-monorepo
+# File: management-changelog.yml
+# Path: \.github\workflows\management-changelog.yml
+# Created Date: Monday, February 19th 2024
+# Author: Jonathan Stevens, jonathan@resnovas.com
+# Github: https://github.com/TGTGamer
+# -----
+# Contributing: Please read through our contributing guidelines.
+# Included are directions for opening issues, coding standards,
+# and notes on development. These can be found at
+# https://github.com/zenstack-monorepo/blob/develop/CONTRIBUTING.md
+# -----
+# Code of Conduct: This project abides by the Contributor Covenant, v2.0
+# Please interact in ways that contribute to an open, welcoming, diverse,
+# inclusive, and healthy community. Our Code of Conduct can be found at
+# https://github.com/zenstack-monorepo/blob/develop/CODE_OF_CONDUCT.md
+# -----
+# Copyright (c) 2024 ZenstackHQ - All Rights Reserved
+# LICENSE: MIT License (MIT)
+# -----
+# This program has been provided under confidence of the copyright holder and
+# is licensed for copying, distribution and modification under the terms
+# of the MIT License (MIT) published as the License,
+# or (at your option) any later version of this license.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+# You should have received a copy of the MIT License
+# along with this program. If not, please write to: jonathan@resnovas.com,
+# or see https://opensource.org/licenses/MIT
+# -----
+# DELETING THIS NOTICE AUTOMATICALLY VOIDS YOUR LICENSE
+###
+
+
+
+on:
+  push:
+    branches:
+    - main # Your main branch
+    - dev # Your development branch
+    - release/* # Your releases branch
+    # - v2 # Your current v2 branch - disabled for now because I don't know if you want to have this tag on this branch or not -_- 
```

Comment:
Yes, I think "release/v2" is a better naming ... However, other contributors are already working off the "v2" branch, so I don't want to cause disturbance. 

The current plan is to continue releasing 2.0.0-alpha and -beta releases from the "v2" branch until a stable one is ready (then merged to main). In this case, we can just add "v2" to the list, right?

Btw, currently "dev" branch serves as an integration branch for features and fixes of the latest major (v1.x.x for now). It's merged to "main" before a release. So "dev" should be removed from the list, I guess?

Reply:
I'll leave this step up to you :) 

I use GitFlow for my Repos, but if you are using TRUNK flow getting rid of Dev makes sense